data/shared/citizen/scripting/lua/scheduler.lua

```diff
@@ -425,23 +402,13 @@ if isDuplicityVersion then
 			cb(0, nil, {}, 'Failure handling HTTP request')
 		end
 	end
-
-	function PerformHttpRequestAwait(url, method, data, headers, options)
```

Comment:
Removing `PerformHttpRequestAwait`?

Reply:
Woops this was added today and I overwrote the file, my bad on that